### PR TITLE
fix bug in validate layout

### PR DESCRIPTION
**Pull request by moishce**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/24439

## Description
fix a bug were edit layout are failed on validate

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Changelog
+* Fixed a bug in the **validate** command failed on "edit layout" files.
```
Comment (ShahafBenYakir) [inline CHANGELOG.md:2]: what is "edit layout" files?

Comment (moishce) [reply CHANGELOG.md:2]: There are several types of layout: _edit_,  _details_ , _close_, and more..

Comment (moishce) [reply CHANGELOG.md:2]: And it fails only in edit files

Comment (ShahafBenYakir) [inline CHANGELOG.md:2]:
```suggestion
* Fixed a bug in the **validate** command for layout files that had `sortValues` fields.
```

**Pull request merged**